modules/ray_tracing/doc/content/sqa/index.md

```diff
@@ -1 +1 @@
-!template load file=sqa/app_index.md.template app=RayTracingApp category=ray_tracing
\ No newline at end of file
+!template load file=sqa/app_index.md.template app=Ray Tracing category=ray_tracing
```

Comment:
Commenting here as an example for my question - it wasn't necessary to use quotes here around `app`? Used to seeing this in apps as simply `FooApp` to connect a template back to proper source rather than as a test replacement.

Reply:
So this argument is only used as a string in the template. Basically the title of the document and or something like "{{app}} is a MOOSE-based application". The `category` is the important one here, which is used to link to the tests.

Reply:
Do you see cases where it is not rendering properly due to this change? I can't find any.

Reply:
You are right; I had to go back to the code and look for other places where the key might be used. Also it looks like there is a bit of regex occurring such that `Ray Tracing` doesn't need to be in quotations. Thanks for clarifying the former for me!